### PR TITLE
Add login name and source id for admin user searching API

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

**Comment by silverwind** [comment]
No swagger schema update?

File: models/user/search.go [hunk 1]
```diff
@@ -62,6 +64,13 @@ func (opts *SearchUserOptions) toSearchQueryBase() *xorm.Session {
 		cond = cond.And(builder.Eq{"id": opts.UID})
 	}
 
+	if opts.SourceID > 0 {
```
Comment (techknowlogick) [inline models/user/search.go:67]: this should only be exposed to admins, as login source seems to be rather sensitive information.

Comment (lunny) [reply models/user/search.go:67]: Done.

**Comment by lunny** [comment]
@techknowlogick It's ready now.

**Review by techknowlogick — APPROVED** [review]

**Review by Zettat123 — APPROVED** [review]

**Review by wolfogre — APPROVED** [review]

**Pull request merged**